lib/chuckt.js

```diff
@@ -108,11 +108,23 @@ window.epixa || (window.epixa = {});
    *
    * @param event
    */
-  ChuckT.prototype.removeListeners = function(event) {
+  ChuckT.prototype.removeListeners = function(event, callback) {
     if (typeof event === 'undefined') {
       this.listeners = {};
     } else {
-      delete this.listeners[event];
+      if (typeof listener !== "undefined") {
```

Comment:
Did you mean `typeof callback` here?

Also, can you update the comment for this function to include the additional argument as well as details for how the argument is used.